app/src/components/v-form/v-form.vue

```diff
@@ -259,7 +259,7 @@ function apply(updates: { [field: string]: any }) {
 		: Object.keys(updates).filter((key) => {
 				const field = fieldsMap.value[key];
 				if (!field) return false;
-				return field.schema?.is_primary_key || !isDisabled(field);
+				return updates.$type === 'created' || field.schema?.is_primary_key || !isDisabled(field);
 		  });
```

Comment:
```suggestion
				return (updates.$type === 'created' && field.meta?.readonly) || field.schema?.is_primary_key || !isDisabled(field);
```

Seeing as the cause of this update being left out is the fact that the `field.meta.readonly === true`, should we maybe add that as an extra layer of assertion? Or is that maybe too specific of a case?

Reply:
I don't see how this relates to the readonly state. What do you mean by that?

Reply:
Thank you for your suggestion :heart: Makes sense 🚀